### ISSUE TITLE
Re-enable PILZ runtime tests

### ISSUE THREAD

**Issue by henningkayser** [ros-planning/moveit2#833]
Related to: https://github.com/ros-planning/moveit/issues/2947
The migration of the planner is completed and all the tests pass with the PRBT (#315).
However, we don't want to get the PRBT robot merged into `moveit_resources`, instead we should fix and enable all tests to use some other robot that still gets maintained.

**Comment by vatanaksoytezer** [comment]
As far as I know the tests are enabled now, can we close this @henningkayser ?

**Comment by henningkayser** [comment]
@vatanaksoytezer not really, only the unit tests are enabled by now. The integration tests are still disabled. @Abishalini is starting to work on this.
